README.md

```diff
@@ -1,5 +1,10 @@
 # Metal3 MariaDB Container
 
+[![CLOMonitor](https://img.shields.io/endpoint?url=https://clomonitor.io/api/projects/cncf/metal3-io/badge)](https://clomonitor.io/projects/cncf/metal3-io)
```

Comment:
mariadb is not one of the clomonitor projects we have listed, does it make sense to have the badge?